fuel/src/main/kotlin/com/github/kittinunf/fuel/toolbox/HttpClient.kt

```diff
@@ -260,6 +276,13 @@ internal class HttpClient(
         Method.DELETE, Method.POST, Method.PUT, Method.PATCH -> connection.doOutput = true
     }
 
+    interface StethoHook {
```

Comment:
I prefer `Hooks`

Reply:
I think you called postConnect too late in the process.